src/pages/my_order.tsx

```diff
@@ -68,18 +70,20 @@ const MyOrder = () => {
           depositRatio: ordersData[order].depositRatio,
           items: ordersData[order].items,
         };
-      });
+      }, updateSignal);
       setMyOrders(myOrdersSpec);
     };
 
     fetchData(setMyOrders);
-  }, []);
+  }, [updateSignal]);
 
   return (
-    <ProfileItemContainer>
-      <MyOrderContainer myOrders={myOrders} />
-      {imgSrc && courtDesign && <PdfCanvas courtDesign={courtDesign} imgSrc={imgSrc as string} />}
-    </ProfileItemContainer>
+    <updateOrderContext.Provider value={[updateSignal, setUpdateSignal]}>
+      <ProfileItemContainer>
+        <MyOrderContainer myOrders={myOrders} />
+        {imgSrc && courtDesign && <PdfCanvas courtDesign={courtDesign} imgSrc={imgSrc as string} />}
+      </ProfileItemContainer>
+    </updateOrderContext.Provider>
```

Comment:
So you wrap some states in a context provider. Have you noticed that there is no context used anywhere in the CC app? We have Redux, we can pass state as props, but we don't have context now.

Unless you are sure what you doing, I cannot suggest you use context here.